config/joshuto.toml

```diff
@@ -12,6 +12,8 @@ show_hidden = false
 show_icons = true
 show_preview = true
 tilde_in_titlebar = true
+# no, absolute, relative
+line_nums = "no"
```

Comment:
I think having something like `show_line_numbers` or something would be better. No need to shorten it as its just a config file that we probably won't be modifying often.